### ISSUE TITLE
Test deployment of Next.js

### ISSUE THREAD

**Issue by waynew99** [ProfessorAmanda/econsimulations#293]
To make sure we don't mess up the current repository when trying to make deployment work, I make a separate repository to experiment with the deployment process. I will update here the progress.

https://github.com/waynew99/econsimstest

**Comment by waynew99** [comment]
Good news! After making several small tweaks, I've successfully deployed on the temporary testing repository. It can be visited at: https://waynew99.github.io/econsimstest.

We can schedule a meeting sometime to live deploy this official repository.

**Comment by waynew99** [comment]
Hi Amanda (@ProfessorAmanda),

I just fixed the deployment issue and successfully deployed the current version!
I also just found a bug loading the data from csv in the deployed website, and made a fix on the master branch. You can try deploying it with the steps below. Please let me know if problems occur!

Steps:
1. Create a new Pull Request from `master` to `deployment`
2. Wait for the GitHub Action checks to finish. (It is shown as yellow progress icon above the "Merge" button on the pull request's page. The "Merge" button will turn grey while the checks are performed)
3. Once the checks pass and turn green, hit the green "Merge" button to merge the pull request into deployment branch. This will trigger a GitHub Pages deployment, which can be seen under the Actions page as a task named "pages build and deployment".
4. Once the deployment task is finished (turns green), the deployment is done and the new version of website can be visited.

**Comment by ProfessorAmanda** [comment]
Thanks, Wayne! I'll give it a whirl!

**Comment by ProfessorAmanda** [comment]
Hi Wayne, sorry to bother you, but something is all screwed up. Do you have any advice?
![Screen Shot 2022-08-03 at 12 18 23 PM](https://user-images.githubusercontent.com/53241468/182658903-4313460a-a51c-4209-bf7c-0d1e59edfa58.png)

**Comment by waynew99** [comment]
Hmmm... sorry that I don't have a solution off the top of my head. I'll do some search and get back to you ASAP!

**Comment by waynew99** [comment]
Shall we try deleting the `package-lock.json` file in the current foler, then running `npm install --legacy-peer-deps` again?

**Comment by ProfessorAmanda** [comment]
Sure. Send me some instructions, and I'll give it a shot!

**Comment by waynew99** [comment]
I think we can try deleting the `package-lock.json` file in the current foler, then running `npm install --legacy-peer-deps` again.

**Comment by ProfessorAmanda** [comment]
You mean, in the online repo, or in my local folder?

Amanda G. Gregg
Associate Professor of Economics, Middlebury College
Join My Personal Zoom Room<https://middlebury.zoom.us/my/agregg?pwd=OWlGMmZMSWJaUkowRG5DUWJtRm9CQT09> (Password: EconHist)
Office: Farrell House 101
Office Phone: (802) 443 - 3419
Pronouns: she/her/hers

________________________________

**Comment by waynew99** [comment]
Oh sorry, I meant the local folder.

**Comment by ProfessorAmanda** [comment]
Got it! Just a sec...

Amanda G. Gregg
Associate Professor of Economics, Middlebury College
Join My Personal Zoom Room<https://middlebury.zoom.us/my/agregg?pwd=OWlGMmZMSWJaUkowRG5DUWJtRm9CQT09> (Password: EconHist)
Office: Farrell House 101
Office Phone: (802) 443 - 3419
Pronouns: she/her/hers

________________________________

**Comment by ProfessorAmanda** [comment]
Boom! Okay I'm going to try deploying now...

**Comment by ProfessorAmanda** [comment]
Animation is back! Nice work :)

**Comment by ProfessorAmanda** [comment]
Awesome! The only page that didn't work for me is Sample Distribution of OLS Estimators (404 not found). It's really great to have separate addresses for each module.

Amanda G. Gregg
Assistant Professor of Economics, Middlebury College
Join My Personal Zoom Room<https://middlebury.zoom.us/my/agregg?pwd=OWlGMmZMSWJaUkowRG5DUWJtRm9CQT09> (Password: EconHist)
Office: Farrell House 101
Office Phone: (802) 443 - 3419
Pronouns: she/her/hers

________________________________
